sdk/containerregistry/azure-containerregistry/CHANGELOG.md

```diff
@@ -1,14 +1,16 @@
 # Release History
 
-## 1.0.0b8 (Unreleased)
+## 1.0.0 (2022-01-19)
 
 ### Features Added
 
-### Breaking Changes
+- Supported passing the rest api version via `ContainerRegistryClient`.
```

Comment:
Is this the phrasing we use elsewhere? I believe in storage it's called the "Service Version", probably best to align on that.

Reply:
Thanks for pointing out, the argument is called "api_version" in python according to the guideline ([link](https://azure.github.io/azure-sdk/python_design.html#specifying-the-service-version)).